theme-release-110.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: October 14, 2022
+    :tags: theme, release
+    :author: Benjamin
+    :location: Malmö, Sweden
+
+.. meta::
+    :description lang=en:
+        Information on sphinx-rtd-theme version 1.1.0
+
+
+Theme release 1.1.0
```

Comment:
```suggestion
Announcing sphinx_rtd_theme 1.1.0
```

This is really not obvious without context, we should be explicit both for SEO and for users.

Reply:
Agreed, I just copied from the former release announcement for consistency.